### PR TITLE
Downgrade fluent-bit to 1.6.10

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Downgrade fluent-bit and prepare v2.1.1-rc.1

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/Chart.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ sources:
   - https://github.com/SumoLogic/sumologic-kubernetes-collection
 dependencies:
   - name: fluent-bit
-    version: 0.15.4
+    version: 0.11.0
```
Comment (perk-sumo) [inline deploy/helm/sumologic/Chart.yaml:16]: Please upgrade the README support matrix as well.

Comment (pmalek-sumo) [reply deploy/helm/sumologic/Chart.yaml:16]: Actually we can use `0.12.1`- it also uses `1.6.10`

https://github.com/fluent/helm-charts/blob/fluent-bit-0.12.1/charts/fluent-bit/Chart.yaml

Comment (sumo-drosiek) [reply deploy/helm/sumologic/Chart.yaml]: changed to `0.12.1`

**Review by pmalek-sumo — APPROVED** [review]

File: deploy/README.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ The following matrix displays the tested package versions for our Helm chart.
 
 Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | FluentD | Fluent Bit | Falco | Metrics Server | Telegraf Operator | Tailing Sidecar Operator
 |:-------- |:-------- |:-------- |:-------- |:-------- |:-------- |:--------  |:--------
-2.1.1 - Latest | 12.3.0 | 1.12.1 | 0.15.4 | 1.7.10 | 5.8.4 | 1.1.5 | 0.3.0
-2.1.0 | 12.3.0 | 1.12.1 | 0.15.1 | 1.7.10 | 5.8.1 | 1.1.5 | 0.3.0
```
Comment (perk-sumo) [inline deploy/README.md:72]: 2.1.0 has the `0.15.1`
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.1.0/deploy/helm/sumologic/Chart.yaml#L16

**Pull request merged**